vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -494,113 +511,130 @@ export default {
     chooseFiles: function () {
       document.getElementById("fileUpload").click();
     },
-    validateForm() {
+    validatePhoneNumber() {
+      const phoneRegex = /^\d{10}$/;
+      if (!phoneRegex.test(this.phoneNumber)) {
+        this.showPhoneValidationError = true;
+      } else {
+        this.showPhoneValidationError = false;
+      }
+    },
+    validateEmailAddress() {
       var emailRegx =
         /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
-      if (
-        this.fullName === "" ||
-        this.email === "" ||
-        this.phoneNumber === "" ||
-        this.fileButtonName == "Upload"
-      ) {
-        this.showValidationError = true;
-      } else if (!this.email.match(emailRegx)) {
+      if (!emailRegx.test(this.email)) {
         this.showEmailValidationError = true;
-      } else if (!this.phoneNumber.match(/[0-9]{10}/)) {
-        this.showPhoneValidationError = true;
       } else {
-        this.showValidationError = false;
-        this.showPhoneValidationError = false;
         this.showEmailValidationError = false;
```

Comment:
Can do whole if...else like below,
`this.showEmailValidationError = !emailRegx.test(this.email)`